### PR TITLE
Super linter

### PR DESCRIPTION
**Pull request by AvshRLev**

### CODE REVIEW

**Comment by AvshRLev** [comment]
[Add eslint](https://trello.com/c/T42mxRDY/39-add-eslint)

**Comment by AvshRLev** [comment]
[Add eslint](https://trello.com/c/T42mxRDY/39-add-eslint)

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "jest",
-    "webpack": "webpack"
+    "webpack": "webpack",
+    "lint": "./node_modules/.bin/eslint ./"
```
Comment (buzzdan) [inline package.json:9]: Is that the recommended way to do it? I think you can use npx instead that does exactly this...

File: .github/workflows/super-linter.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  pull_request:
+    branches: [master, main]
+
+###############
+# Set the Job #
+###############
+jobs:
+  build:
+    # Name the Job
+    name: Lint Code Base
```
Comment (buzzdan) [inline .github/workflows/super-linter.yml:24]: I think its very confusing to name the job and github action with the same name

File: .github/workflows/super-linter.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
+
+      ################################
+      # Run Linter against code base #
+      ################################
+      - name: Lint Code Base
+        uses: github/super-linter@v3
+        env:
+          VALIDATE_ALL_CODEBASE: false
```
Comment (buzzdan) [inline .github/workflows/super-linter.yml:47]: No python/Django validation for now?

**Comment by buzzdan** [comment]
You must have a way to run super linter locally...
And, please add the lint task also to your makefile

File: .github/workflows/test-run.yml [hunk 4]
```diff
@@ -8,7 +8,7 @@ jobs:
       - uses: actions/setup-node@v1
       - run: pip install django
       - run: npm ci
-      - run: make all & sleep 10 && npm test
+      - run: make run & sleep 10 && npm test
```
Comment (buzzdan) [inline .github/workflows/test-run.yml:11]: just use `make test` instead

File: Makefile [hunk 5]
```diff
@@ -1,3 +1,11 @@
-all:
-	npm run webpack
-	python manage.py runserver 
\ No newline at end of file
+run:
```
Comment (buzzdan) [inline Makefile:1]: add:

```make
all: lint test

```

in first line

**Pull request merged**